common/plugins/eu.esdihumboldt.hale.common.filter/src/eu/esdihumboldt/hale/common/filter/internal/EntityReplacementVisitor.java

```diff
@@ -78,8 +78,7 @@ private String toPropertyName(EntityDefinition entityDefinition) {
 					return c.getChild().getName().getLocalPart();
 					// TODO use version w/ namespace in ambiguous cases?
 				}).collect(Collectors.joining("."));
-		// put in quotes so it is possible to parse the resulting filterTerm
-		return '"' + name + '"';
+		return name;
```

Comment:
So this is no longer necessary? Did it cause problems?

Reply:
No, it's no longer necessary as far as I can see. The filter parser in GeoTools was changed and now correctly parses the name without quotes. With the quotes, the parser now escapes them, e.g. `"aa" = 'test'` is now parsed as `""aa"" = 'test'`.